### ISSUE TITLE
Cypress custom commands doesn't load

### ISSUE THREAD

**Issue by nghiahung-dev** [remix-run/indie-stack#251]
### Have you experienced this bug with the latest version of the template?

Yes

### Steps to Reproduce

Run cypress test in local
`npm run test:e2e:dev`

### Expected Behavior

We can run cypress test successfully

### Actual Behavior

```
cy.cleanupUser is not a function
Because this error occurred during a after each hook we are skipping the remaining tests in the current suite: smoke tests
```

![Screen Shot 2023-08-23 at 11 10 09](https://github.com/remix-run/indie-stack/assets/52577080/2c3c7f03-718b-454b-a747-22c77ec27f3c)

**Comment by ngtrthinh169TPC** [comment]
I'm having the same problem with `cy.cleanupUser is not a function`.
![image](https://github.com/remix-run/indie-stack/assets/74962522/62fc3341-a5f0-4103-9e26-9fcb9d129bef)

 Weirdly I'm also seeing the same problem with the grunge stack yesterday. Thinking there might be something between Cypress and Remix setups in general, not only this stack.

**Comment by MichaelDeBoey** [comment]
This seems to be a regression in Cypress with using `Cypress.Command.add`

Fix in #254
